resources/assets/css/_user_tagger.css

```
@@ -0,0 +1,20 @@
+/* purgecss start ignore */
+.UserTagger .highlighted .UserTagger--username {
```

```suggestion
.ark-user-tagger .highlighted .ark-user-tagger--username {
```

I meant something like the above instead @alfonsobries , as we generally don't use PascalCase

---

@ItsANameToo ✅